src/init/replace_vulnerability_detector.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
+
+# Copyright (C) 2015, Wazuh Inc.
+# All rights reserved.
+# Wazuh.com
+
+# This program is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation.
+
+# Shell script that updates 'Vulnerability Detector' to 'Vulnerability Detection'
+
+# Function: isConfigPresent
+# Description: Function that checks if a 'configuration' is present in the 'ossec.conf' file.
+# Parameters:
+#   $1: Path to the ossec.conf file.
+#   $2: Configuration tag.
+isConfigPresent()
+{
+    local OSSEC_CONF_PATH="$1"
+    local CONFIG_PATTERN="$2"
+
+    if ( grep -q "$CONFIG_PATTERN" "$OSSEC_CONF_PATH" ); then
+        return 0
+    fi
+
+    return 1
+}
+
+# Function: addWodleConfig
```

Comment:
Question: is `wodle` the right name here? There are some 'wodles' in the config but not every config section is a wodle: https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/index.html